media/plg_system_tour/js/tours.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    $(document).ready(function() {
```

Comment:
@Hutchy68 @mbabker https://github.com/linkedin/hopscotch
Hopscotch as well is jQuery free so...